### PR TITLE
clippy and fmt changes

### PR DESCRIPTION
**Pull request by sn99**
I was using your crate to make a [wrapper library](https://github.com/SubconsciousCompute/query-wmi) and thought about migrating to`windows-rs` from `winapi`. This pull is precursor to it.

Although this test seems to be failing even before I made any changes

```rust
test de::wbem_class_de::tests::it_desr_into_map ... FAILED

failures:

---- de::wbem_class_de::tests::it_desr_into_map stdout ----
thread 'de::wbem_class_de::tests::it_desr_into_map' panicked at 'assertion failed: langs.contains(&Variant::String(system_lang))', src\de\wbem_class_de.rs:302:42
```

### CODE REVIEW

File: src/datetime.rs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 use crate::WMIError;
-use std::{str::FromStr, fmt};
 use chrono::prelude::*;
 use serde::{de, ser};
+use std::{fmt, str::FromStr};
 
 /// A wrapper type around `chrono`'s `DateTime` (if the `chrono` feature is active. ), which supports parsing from WMI-format strings.
 #[derive(Debug)]
-pub struct WMIDateTime(pub chrono::DateTime<FixedOffset>);
+pub struct WMIDateTime(pub DateTime<FixedOffset>);
```
Comment (ohadravid) [inline src/datetime.rs:8]: Seems like `FixedOffset::east` was deprecated? Mind fixing the warning?

**Review by ohadravid — APPROVED** [review]

**Comment by ohadravid** [comment]
Thanks @sn99 😁 

You wrapper is a very nice idea! 
Defining the actual WMI objects in Rust is finicky, but IMO for a lot of usecases just a `HashMap<String, Variant>` is not very useful, so maybe consider an API which could be later extended to use automatic MOF/MSDN format -> Rust fields conversion, and maybe support for a user-provided connection?

I think migrating to `windows-rs` would be really awesome.
If you continue working on it, try to have the work split into manageable PRs?

**Comment by sn99** [comment]
That's the plan! I've already begun working on a separate PR to migrate to `windows-rs`. Additionally, I'm looking to expand the appeal of the wrapper to a wider audience (currently, I've only proposed using it for office work instead of `sysinfo`). I'll revisit this PR in about a week or so, after my university midterms are over. Thanks!

**Comment by ohadravid** [comment]
FYI #67 / @vthib

**Comment by vthib** [comment]
Interesting ^^

I already have a working migration to `windows` locally, I just have to clean-it up before making a PR for it, so you may not need to spend too much time doing this migration yourself @sn99

**Comment by sn99** [comment]
@vthib I will leave it to you then

**Comment by sn99** [comment]
Closed in favor of #67

**Pull request closed without merging**